includes/TinyMCE.php

```diff
@@ -98,6 +97,12 @@ public function web_stories_mce_plugin( array $plugins ) {
 	 */
 	public function web_stories_tinymce_scripts() {
 		$this->enqueue_style( 'wp-components' );
```

Comment:
```suggestion

```
This line can now be removed.

Reply:
So does  WP automatically load the wp-components _CSS_ when using the wp-components JS?

Reply:
🤦 Style, no it is needed.